Lib/sqlite3/test/userfunctions.py

```diff
@@ -2,7 +2,7 @@
 # pysqlite2/test/userfunctions.py: tests for user-defined functions and
 #                                  aggregates.
 #
-# Copyright (C) 2005-2007 Gerhard H�ring <gh@ghaering.de>
+# Copyright (C) 2005-2007 Gerhard Häring <gh@ghaering.de>
```

Comment:
It seems like the UTF-8 issue was reintroduced.

Reply:
That's very strange; I can't find that change in my repo. If it's ok with you, I'll re-encode the file with UTF-8.

Reply:
Aha, it was @berkerpeksag's commit (edited with GitHub; same problem as yesterday).